marlowe-playground-client/test/Marlowe/ParserTests.purs

```diff
@@ -1,55 +1,31 @@
 module Marlowe.ParserTests where
 
 import Prelude
-import Control.Alternative ((<|>))
 import Control.Monad.Reader (runReaderT)
 import Data.Either (Either(..))
 import Data.Maybe (fromMaybe)
 import Data.String (Pattern(..), stripPrefix, stripSuffix, trim)
-import Effect.Aff (Aff)
-import Foreign (fail)
-import Marlowe.Gen (genAction, genContract, genObservation, genTransactionWarning, genValue)
+import Marlowe.Gen (genContract)
 import Marlowe.GenWithHoles (GenWithHoles, unGenWithHoles)
-import Marlowe.Holes (Action, Contract, Observation, Value)
-import Marlowe.Parser (action, observation, parseContract, transactionWarning, value)
-import Marlowe.Semantics (TransactionWarning)
+import Marlowe.Holes (Contract)
+import Marlowe.Parser (parseContract)
 import Test.QuickCheck (class Testable, Result, (===))
 import Test.Unit (Test, TestSuite, failure, success, suite, test)
 import Test.Unit.QuickCheck (quickCheck)
 import Text.Parsing.StringParser (runParser)
-import Text.Parsing.StringParser.Basic (integral, parens)
+import Text.Parsing.StringParser.Basic (integral)
 import Text.Pretty (genericPretty)
 
 all :: TestSuite
 all =
   suite "Marlowe.Parser" do
     test "Numbers Parser" $ integralParser
```

Comment:
is the integralParser still used somewhere?

Reply:
nope... removed